components/org.wso2.carbon.identity.local.auth.smsotp.event.handler/src/test/java/org/wso2/carbon/identity/local/auth/smsotp/event/handler/notification/TestableSMSNotificationHandler.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.local.auth.smsotp.event.handler.notification;
+
+import org.wso2.carbon.identity.event.event.Event;
+
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Moved class from https://github.com/wso2-extensions/identity-local-auth-smsotp/pull/25/files#r1829290970. 
Renamed for better reflect the purpose of the implementation.